activiti-services/activiti-services-rest/src/main/java/org/activiti/services/rest/controllers/ProcessDefinitionController.java

```diff
@@ -73,4 +95,23 @@ public ProcessDefinitionResource getProcessDefinition(@PathVariable String id) {
         }
         return resourceAssembler.toResource(processDefinitionConverter.from(processDefinition));
     }
+
+    @RequestMapping(value = "/{id}/xml",
+            method = RequestMethod.GET)
+    public ProcessModelResource getProcessModel(@PathVariable String id) {
+        try (final InputStream resourceStream = repositoryService.getProcessModel(id)) {
+            String xml = new String(IoUtil.readInputStream(resourceStream, null), StandardCharsets.UTF_8);
+            return processModelResourceAssembler.toResource(new ProcessModel(id, xml));
+        } catch (IOException e) {
+            throw new ActivitiException("Error occured while getting process model '" + id + "' : " + e.getMessage(),
+                                        e);
+        }
+    }
+
+    @RequestMapping(value = "/{id}/json",
+            method = RequestMethod.GET)
+    public BpmnModelResource getBpmnModel(@PathVariable String id) {
+        BpmnModel bpmnModel = repositoryService.getBpmnModel(id);
+        return bpmnModelResourceAssembler.toResource(new MetaBpmnModel(id, bpmnModel));
```

Comment:
@daisuke-yoshimoto same here, just return the json content of the process definition. In the same way that we are going to return the svg content when we have it.